pages/common/hping.md

```diff
@@ -0,0 +1,24 @@
+# hping
+
+> Hping is a command-line oriented TCP/IP packet assembler/analyzer. The interface is inspired by the ping UNIX command.
+> More information: <http://www.hping.org>.
+
+- Make a ping with TCP protocol on localhost:
+
+`hping3 {{localhost}}`
+
+- Make a ping with TCP on 127.0.0.1 address on port 80:
```

Comment:
Perhaps this will read better?
```suggestion
- Ping over TCP on a specific port:
```

Reply:
I propose to use the format you defined in the last comment:
`Ping an IP address over TCP on a specific port`

Reply:
Sounds good to me! There are only suggestions after all.